### ISSUE TITLE
Implementing the coding standard on the web\server.py file

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7949]
|<img alt="vladius's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| vladius reported|
|-|-|
|Trac ID|trac#7949|
|Type|enhancement|
|Created|2015-06-22 21:08:22Z|
|Branch|https://github.com/twisted/twisted/tree/t-w-server-cleanup-7949|

The `web\server.py` doesn't adhere to the [coding guidelines](https://twistedmatrix.com/documents/current/core/development/policy/coding-standard.html) now.

Attachments:

* [server.py-cleanup.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/7c7/7c7aa174b39f1326ce6ac53d2ea9a47967875e1d/6f5e9a29b90346a3b27b69e335025bfb6860a820.patch) (8823 bytes) - added by vladius on 2015-06-22 21:14:35Z - The patch to fix the coding style.
* [server.py-cleanup.2.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/7c7/7c7aa174b39f1326ce6ac53d2ea9a47967875e1d/ee37e43262b1564d1fa82bbd05c896398463cc52.patch) (9002 bytes) - added by vladius on 2015-06-22 21:38:07Z - The patch to fix the coding style.

<details><summary>Searchable metadata</summary>

```
trac-id__7949 7949
type__enhancement enhancement
reporter__vladius vladius
priority__low low
milestone__ 
branch__branches_t_w_server_cleanup_7949 branches/t-w-server-cleanup-7949
branch_author__adiroiban adiroiban
status__closed closed
resolution__fixed fixed
component__web web
keywords__ 
time__1435007302677201 1435007302677201
changetime__1435057095767589 1435057095767589
version__None None
owner__vladius vladius

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="vladius's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_1"></a>|vladius removed owner|
|-|-|

I have implemented the coding standard on a `web\server.py` file. Created removed the spaces around named argument, added two lines between class methods, made the code adhere to the line margin.

I've used PEP8 for minor whitespace-formatting issues.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_2"></a>|@adiroiban commented|
|-|-|

(In [45105]) Branching to t-w-server-cleanup-7949.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set owner to vladius|
|-|-|

Changes look good. Thanks!

We dont use '\' for breaking long line. I have updated the patch to not use them.

I am sending your patch to the buildbot and if all is ok I will merge. Thanks!

**Comment by twisted-trac** [comment]
|<img alt="vladius's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|vladius commented|
|-|-|

Replying to [adiroiban](#note_3):
> We dont use '\' for breaking long line.

Is it stated in the coding standard? I don't remember getting this as a warning while running the twistedchecker tool...

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_5"></a>|@adiroiban set status to `closed`|
|-|-|

(In [45109]) Merge t-w-server-cleanup-7949: Fix some coding standard errors in t.w.server

Author: vladius
Reviewer: adiroiban
Fixes: [#7949](https://github.com/twisted/twisted/issues/7949)
